app/main.py

```diff
@@ -1,13 +1,16 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {"line": error.get("line_number"),
+            "column": error.get("column_number"),
+            "message": error.get("text"), "name": error.get("code"),
+            "source": "flake8"}
```

Comment:
```suggestion
    return {
        "line": error.get("line_number"),
        "column": error.get("column_number"),
        "message": error.get("text"), 
        "name": error.get("code"),
        "source": "flake8"
    }
```
Please improve readability